src/db/users.rs

```diff
@@ -1,8 +1,9 @@
+use log::{info, warn};
```

Comment:
Please use the logging functions from `tracing` instead. You can just call `tracing::info!` or `tracing::log!` and it should work.